StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -95,6 +97,19 @@ class RowButton: UIView {
         } else {
             self.sublabel = nil
         }
+
+        if showDefaultPMBadge {
+            let defaultBadge = UILabel()
```

Comment:
Nit: I know you're following the `sublabel` pattern, but might be cleaner if you followed the `checkmarkImageView` pattern, up to you.